src/libs/pool-data-provider/hooks/use-incentives-data-context.tsx

```diff
@@ -62,7 +62,9 @@ export function IncentivesDataProvider({ children }: { children: ReactNode }) {
     currentAccount,
     networkConfig.addresses.chainlinkFeedRegistry,
     networkConfig.usdMarket ? Denominations.usd : Denominations.eth,
-    preferredConnectionMode === ConnectionMode.rpc || network !== apolloClientNetwork
+    preferredConnectionMode === ConnectionMode.rpc ||
+      network !== apolloClientNetwork ||
+      !networkConfig.addresses.uiIncentiveDataProvider
```

Comment:
not happy with that fix, when i have some time i'll look into redoing the dataflow.
It's kinda nutzs that we go soo deep down the dom just to set a boolean skip flag because there's no address set